### ISSUE TITLE
Space Between option does not work well in countdown widget (only in RTL)

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#21503]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi

As you can see in the image below, if we increase the Space Between value in an RTL website, the middle space will increase twice as much as the other spaces.

![countdown](https://user-images.githubusercontent.com/77163248/222280368-642ed889-5383-4336-980d-a6a80b0a3268.png)

### Steps to reproduce

1- Make sure the website is RTL.
2- Add a countdown widget.
3- In the Style tab of this widget, increase the Space Between value.

Now if you look carefully at the space created between the time of the clock and exactly, it is double the other spaces. If you change the website to LTR, the problem will be solved.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

[system-info-localhost-2-03-2023.txt](https://github.com/elementor/elementor/files/10865763/system-info-localhost-2-03-2023.txt)

**Comment by nicholaszein** [comment]
Hello @79ho3ein! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by rami-elementor** [comment]
Hi @79ho3ein 

I can't recreate the issue. It looks ok on my testing sites, both LTR and RTL sites.

According to the system info, you tested with v3.11, maybe it was already fixed? can you confirm that the issue still exist?

**Comment by 79ho3ein** [comment]
Hi @rami-elementor,

I'm still facing this bug on version 3.14.

**Comment by rami-elementor** [comment]
@79ho3ein I can't recreate on my environment, maybe you have custom CSS on the widgets?

**Comment by 79ho3ein** [comment]
@rami-elementor No, I don't use any css code. But I think I found the cause of the problem.

This seems to be where the problem comes from.

![1](https://github.com/elementor/elementor/assets/77163248/e2501722-7e52-4109-8c12-ed15e5bffc2a)

In LTR, the flex direction is in row and there is no problem. But in RTL, Elementor converts it to row-reversed which causes this problem. On the other hand, row-reverse helps to convert the widget to RTL. Not sure if there is another solution. Please watch the video below.

https://github.com/elementor/elementor/assets/77163248/0c7ac9cd-5962-401a-bed3-910b30772cd1

**Comment by rami-elementor** [comment]
@79ho3ein Watch the video again, changing `flex-direction` to `row` in RTL view switches the order of the elements, moving the days from the left side to the right side. It has the potential to break existing websites.

P.S. I was thinking using `gap` on the parent element instead, but [Safari started supporting flex gap in Safari 14.1](https://caniuse.com/flexbox-gap) while [Elementor supports Safari 14.0 and above](https://elementor.com/help/requirements/) so we can't use it.

**Comment by 79ho3ein** [comment]
@rami-elementor, Right, so nothing can be done about it now.

**Comment by nicholaszein** [comment]
> Right, so nothing can be done about it now.

Not for the time being @79ho3ein, but we will keep an eye on potential solutions for this issue that are backward compatible.

**Comment by rami-elementor** [comment]
I'm planing to add a new "Direction" control. This may help solve the issue and provide a new capability to control the elements direction.

**Comment by 79ho3ein** [comment]
@rami-elementor, How does the direction option work in this situation? Doesn't this mess up the order of the options(days and...)?
